documentation/modules/configuring/con-config-kafka.adoc

```diff
@@ -21,18 +21,14 @@ Configuration options that are particularly important include the following:
 * Rack awareness
 * Metrics
 * Cruise Control for cluster rebalancing
+* Metadata version for KRaft-based Kafka clusters
+* Inter-broker protocol version for ZooKeeper-based Kafka clusters
 
-For a deeper understanding of the Kafka cluster configuration options, refer to the link:{BookURLConfiguring}[Strimzi Custom Resource API Reference^].
-
-.Kafka versions
-
-The `inter.broker.protocol.version` property for the Kafka `config` must be the version supported by the specified Kafka version (`spec.kafka.version`).
-The property represents the version of Kafka protocol used in a Kafka cluster.
+The `metadataVersion` or `inter.broker.protocol.version` property for the Kafka `config` must be a version supported by the specified Kafka version (`spec.kafka.version`).
```

Comment:
Can we make it more clear here that `metadataVersion` is a separate field in `.spec.kafka` but `inter.broker.protocol.version` is a field in the `.spec.kafka.config`? I think it will be confusing the way it is written now (as many users might be confused the metadata version is not in the Kafka config anymore and might misread it). Maybe like this?

```suggestion
The `.spec.kafka.metadataVersion` field or `inter.broker.protocol.version` property in `.spec.kafka.config` must be a version supported by the specified Kafka version (`spec.kafka.version`).
```